Code/Framework/AzToolsFramework/AzToolsFramework/Entity/PrefabEditorEntityOwnershipInterface.h

```diff
@@ -19,11 +19,31 @@
 
 namespace AzToolsFramework
 {
+    namespace Prefab
+    {
+        //! A RootAliasPath can be used to store an alias path that starts from the Prefab EOS root instance.
+        //! The root instance itself is included in the path. These can be used as Instance handles across systems
+        //! that do not have visibility over InstanceOptionalReferences, or that need to store Instance handles
+        //! for longer than just the span of a function without the risk of them going out of scope.
+        using RootAliasPath = AliasPath;
+    }
 
     class PrefabEditorEntityOwnershipInterface
     {
     public:
-        AZ_RTTI(PrefabEditorEntityOwnershipInterface,"{38E764BA-A089-49F3-848F-46018822CE2E}");
+        AZ_RTTI(PrefabEditorEntityOwnershipInterface, "{38E764BA-A089-49F3-848F-46018822CE2E}");
+
+        //! Returns whether the system has a root instance assigned.
+        //! /return True if a root prefab is assigned, false otherwise.
```

Comment:
nit: I think this should be `\return` (or even better `@return`) 🙂

Reply:
Oh that's good, I thought it would be `@` but I saw the rest of the file used `/` and just conformed. Will fix everywhere!